frontend/app/utils/zod-utils.ts

```diff
@@ -34,3 +36,11 @@ export function transformFlattenedError<T extends { fieldErrors: Record<string,
   const transformedEntries = Object.entries(flattenedError.fieldErrors).map(([key, value]) => [key, value.at(0)]);
   return Object.fromEntries(transformedEntries);
 }
+
+type FlattenedZodError = typeToFlattenedError<ZodError<string>>;
+
+export function getErrorList(flattenedError: FlattenedZodError) {
+  const transformedEntries = flattenedError.fieldErrors.errors ?? [];
+  const transformedEntries2 = flattenedError.formErrors;
```

Comment:
This might be a bit nitpicky.. but are there alternative names for these that better describe how they're different? Naming these Thing1 and Thing2 confused me. (but that might be a *me problem*)

Reply:
Maybe even something as simple as:

``` typescript
export function getErrorList(flattenedError: FlattenedZodError) {
  const fieldErrors = flattenedError.fieldErrors.errors ?? [];
  const formErrors = flattenedError.formErrors;
  return [...fieldErrors, ...formErrors];
}
```

Reply:
Error summary is more complicated than displaying errors in a list since they have to be links to actual HTML inputs.